web/src/beta/lib/reearth-ui/components/Icon/icons.ts

```diff
@@ -0,0 +1,233 @@
+import A_Z from "./Icons/A-Z.svg?react";
+import AddStyle from "./Icons/AddStyle.svg?react";
+import Appearance from "./Icons/Apperance.svg?react";
+import Article from "./Icons/Article.svg?react";
+import Book from "./Icons/Book.svg?react";
+import BracketsCurly from "./Icons/BracketsCurly.svg?react";
+import Buildings from "./Icons/Buildings.svg?react";
+import CalendarBlank from "./Icons/CalendarBlank.svg?react";
+import Camera from "./Icons/Camera.svg?react";
+import Capture from "./Icons/Capture.svg?react";
+import CaretDoubleDown from "./Icons/CaretDoubleDown.svg?react";
+import CaretDoubleLeft from "./Icons/CaretDoubleLeft.svg?react";
+import CaretDoubleRight from "./Icons/CaretDoubleRight.svg?react";
+import CaretDown from "./Icons/CaretDown.svg?react";
+import CaretLeft from "./Icons/CaretLeft.svg?react";
+import CaretUp from "./Icons/CaretUp.svg?react";
+import Check from "./Icons/Check.svg?react";
+import Checked from "./Icons/Checked.svg?react";
+import Circle from "./Icons/Circle.svg?react";
+import CirclesThreePlus from "./Icons/CirclesThreePlus.svg?react";
+import Clock from "./Icons/Clock.svg?react";
+import ClockAfternoon from "./Icons/ClockAfternoon.svg?react";
+import ClockClockwise from "./Icons/ClockClockwise.svg?react";
+import Close from "./Icons/Close.svg?react";
+import Copy from "./Icons/Copy.svg?react";
+import Cube from "./Icons/Cube.svg?react";
+import Cursor from "./Icons/Cursor.svg?react";
+import Cylinder from "./Icons/Cylinder.svg?react";
+import Dashboard from "./Icons/Dashboard.svg?react";
+import Data from "./Icons/Data.svg?react";
+import Desktop from "./Icons/Desktop.svg?react";
+import DeviceMobile from "./Icons/DeviceMobile.svg?react";
+import DotsSixVertical from "./Icons/DotsSixVertical.svg?react";
+import DotsThreeVertical from "./Icons/DotsThreeVertical.svg?react";
+import EditMode from "./Icons/Edit-Mode.svg?react";
+import Edit from "./Icons/Edit.svg?react";
+import Editor from "./Icons/Editor.svg?react";
+import Exit from "./Icons/Exit.svg?react";
+import Extrude from "./Icons/Extrude.svg?react";
+import Eye from "./Icons/Eye.svg?react";
+import EyeSlash from "./Icons/EyeSlash.svg?react";
+import File from "./Icons/File.svg?react";
+import FileHtml from "./Icons/FileHtml.svg?react";
+import FileMD from "./Icons/FileMD.svg?react";
+import FillCamera from "./Icons/Fill_Camera.svg?react";
+import FillExit from "./Icons/Fill_Exit.svg?react";
+import FillFile from "./Icons/Fill_File.svg?react";
+import FillFileMD from "./Icons/Fill_FileMD.svg?react";
+import FillImage from "./Icons/Fill_Image.svg?react";
+import FillLayer from "./Icons/Fill_Layer.svg?react";
+import FillListDashes from "./Icons/Fill_ListDashes.svg?react";
+import FillPen from "./Icons/Fill_Pen.svg?react";
+import FillSetting from "./Icons/Fill_Setting.svg?react";
+import FillTime from "./Icons/Fill_Time.svg?react";
+import FillVideo from "./Icons/Fill_Video.svg?react";
```

Comment:
For filled icons i suggest naming them as `XXXFilled` so that they can be listed beside the origin version when imports been sorted with alphabet.
For example
```
import Camera from "./Icons/Camera.svg?react";
import CameraFilled from "./Icons/CameraFilled.svg?react";
```

Also could you rename the svg files to avoid `_` or `-` and make sure all naming matches the same format like the first letter is in Uppercase?
For example
```
Fill_Camera.svg -> CameraFilled.svg
A-Z.svg -> AToZ.svg (also imported as AToZ instead of A-Z)
if.svg -> If.svg
Square.svg and Square2.svg -> I'm not sure what's the difference actually, could you try find a more meanful name for them?
```